scripts/test.sh

```diff
@@ -11,7 +11,7 @@ fi
 
 package="community.flock.generated"
 
-languages=("Java" "Kotlin" "Scala" "TypeScript" "Wirespec")
+languages=("Java" "Kotlin" "TypeScript" "Python" "Wirespec" "OpenAPIV2" "OpenAPIV3")
```

Comment:
I don't think OpenAPI should be here as a language per se. Unless we test if it is valid.

Reply:
I agree. I'll make sure to tune Junie to not let this happen again.